lib/ts/supertokens.ts

```diff
@@ -293,6 +306,7 @@ export default class SuperTokens {
         userId: string;
         userIdType?: "SUPERTOKENS" | "EXTERNAL" | "ANY";
         externalUserIdInfo?: string;
+        userContext: Record<string, any>;
```

Comment:
These are not optional (+ they come after an optional param so not sure how that builds/passes linting)